button/popup.css

```diff
@@ -56,25 +56,29 @@ http://matthewlein.com/ceaser/
 .footnote {
     width: 100%;
     height: 55px;
-    bottom: 0;
     color: #959595;
     padding-left: 15px;
     line-height: 57px;
     background: #fff;
+    float: left;
     cursor: default;
     border-top: 1px solid #e3e3e3;
 }
 
 .site { /* each note area */
     height: 50px; /* prev hieght 40px */
-    padding: 2px 0 2px 0;
+    margin: 2px 0px 0px 0px;
     border-top: 1px solid #e3e3e3;
     border-bottom: 1px solid #e3e3e3;
     position: relative;
     z-index: 1;
     width: 100%;
     cursor: pointer;
+}
 
+.site:last-child {
+    border-bottom: 0px;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/2651364002/issues/source?bid=2884821&fileBranchId=3324709#l80)